.github/workflows/on-pr.yml

```diff
@@ -106,7 +106,7 @@ jobs:
           || ''
         }}
       # We'll only upload coverage artifacts with the periodic-coverage cron workflow.
```

Comment:
If this is meant to be merged, this comment is outdated.

Reply:
> Note: disregard CI changes. will revert prior to merge.

Reply:
This comment is outdated, but outside the scope of this PR.